disnake/client.py

```diff
@@ -741,22 +742,39 @@ async def login(self, token: str) -> None:
         data = await self.http.static_login(token.strip())
         self._connection.user = ClientUser(state=self._connection, data=data)
 
-    async def connect(self, *, reconnect: bool = True) -> None:
+    async def connect(
+        self, *, reconnect: bool = True, ignore_session_start_limit: bool = False
+    ) -> None:
         """|coro|
 
         Creates a websocket connection and lets the websocket listen
         to messages from Discord. This is a loop that runs the entire
         event system and miscellaneous aspects of the library. Control
         is not resumed until the WebSocket connection is terminated.
 
+        If the client doesn't have enough connects remaining in the current 24-hour window and ``ignore_session_start_limit`` is ``False``,
```

Comment:
This should probably be moved down to the `Raises` section

Reply:
https://github.com/DisnakeDev/disnake/pull/537/commits/aa0b1bef7c72aff683acd558e03247e004b7f758